(docs): Add Information about how to use Dark Mode SplashScreens (#286)
apache/cordova-plugin-splashscreen: README.md

| xxhdpi  | 960x1600  | 1600x960  |
| xxxhdpi | 1280x1920 | 1920x1280 |

#### Dark Mode (API 28+)

> You can easily provide an extra SplashScreen image for NightMode/DarkMode enabled devices.

Reviewer: ```suggestion
You can optionally provide an extra SplashScreen image to be used in dark/night mode when enabled on supported devices.
```

> To do this, add the keyword `-night` in between the image's **layout** and **size** keywords of the `density` attribute value. E.g.: `land-night-hdpi`

Reviewer: ```suggestion
To do this, add the `-night` keyword in between the **layout** and **size** keywords of the image's `density` attribute value. E.g.: `land-night-hdpi`
```

    <splash src="res/screen/ios/Default@2x~ipad~comany.png" />
```

##### Dark Mode

> Since [Cordova-ios@6.1.0](https://github.com/apache/cordova-ios) it is now possible to use optionally different splashscreen images when your app is running in dark mode. You can configure these images in config.xml with the ~dark suffix (and ~light is also supported).

Reviewer: ```suggestion
Since [Cordova-ios@6.1.0](https://github.com/apache/cordova-ios) it is now possible to optionally specify different SplashScreen images to be used when your app is running in dark mode. You can specify the luminosity of SplashScreen images in config.xml using the ~dark and ~light suffixes.
```

> Since [Cordova-ios@6.1.0](https://github.com/apache/cordova-ios) it is now possible to optionally specify different SplashScreen images to be used when your app is running in dark mode. You can specify the luminosity of SplashScreen images in config.xml using the ~dark and ~light suffixes.

Reviewer 2: ```suggestion
Since [Cordova-iOS@6.1.0](https://github.com/apache/cordova-ios), it is now possible to optionally specify different SplashScreen images to use when the app is running in dark mode. The luminosity of SplashScreen images can be defined in `config.xml` using the `~dark` and `~light` suffixes.
```

```xml
<!-- Default image to be used for all modes -->
<splash src="res/screen/ios/Default@2x~universal~anyany.png" />

<!-- Image to use specifically for dark mode devices -->
<splash src="res/screen/ios/Default@2x~universal~anyany~dark.png" />

<!-- Image to use specifically for light mode devices -->
<splash src="res/screen/ios/Default@2x~universal~anyany~light.png" />
```

> **Note:** This works since iOS13+. Version below will use the default SplashScreen without a luminosity suffix specified.

Reviewer: ```suggestion
**Note:** This works since iOS 13. iOS 12 and below will use the default SplashScreen without a luminosity suffix specified.
```
